yarn.lock

```diff
@@ -1251,7 +1251,7 @@
   resolved "https://registry.yarnpkg.com/@xtuc/long/-/long-4.2.2.tgz#d291c6a4e97989b5c61d9acf396ae4fe133a718d"
   integrity sha512-NuHqBY1PB/D8xU6s/thBgOAiAP7HOYDQ32+BFZILJ8ivkUkAHQnWfn6WhL79Owj1qmUnoN/YPhktdIoucipkAQ==
 
-"@zeit/webpack-asset-relocator-loader@0.5.1":
+"@zeit/webpack-asset-relocator-loader@0.5.2":
   version "0.5.1"
   resolved "https://registry.yarnpkg.com/@zeit/webpack-asset-relocator-loader/-/webpack-asset-relocator-loader-0.5.1.tgz#796e454c18fefa1b08be322c7908e8fa2294ea3e"
```

Comment:
This is the wrong sha, right?

Reply:
Weird, when I did `yarn install` it acted like this was fine.

Reply:
Try `yarn add `@zeit/webpack-asset-relocator-loader@0.5.2`